pyproject.toml

```diff
@@ -17,16 +17,17 @@ dependencies = [
   "bmipy",
   "numpy",
   "xarray",
+  "netcdf4",
```

Comment:
Hmm, we have the container to isolate the bmi model dependencies from other models and ewatercycle package.
Bmi models that require a specific old version of a dependency could give conflicts or restrict an upgrade path.

I expected that `pip install ewatercycle-plugin` would give you `from ewatercycle.models import LeakyBucket` class and all the dependencies needed to make that import work like ewatercycle package,
but now it installs the models dependencies.

I think we should have an optional dependency list called `model` or `bmi` with the bmi model dependencies. These should be installed in the Dockerfile and when working on the bmi class.

Reply:
I refactored all the BMI stuff to a separate repo. This one only has the generic guide from grpc4bmi container -> plugin